--- VST_SDK/VST2_SDK/pluginterfaces/vst2.x/aeffect.h	2017-11-08 07:49:41.000000000 -0500
+++ patches/aeffect.h	2018-01-23 22:50:10.329253858 -0500
@@ -66,7 +66,11 @@
 	#pragma options push -a8
 #elif defined(__GNUC__)
     #pragma pack(push,8)
+#if defined(WIN32)
     #define VSTCALLBACK __cdecl
+#else
+    #define VSTCALLBACK
+#endif
 #elif defined(WIN32) || defined(__FLAT__) || defined CBUILDER
 	#pragma pack(push)
 	#pragma pack(8)
